tests/typings/chai-samsam.d.ts

```diff
@@ -0,0 +1,6 @@
+/* tslint:disable:no-namespace */
```

Comment:
I think this means our TSLint settings are wrong. According to https://palantir.github.io/tslint/rules/no-namespace/ we should be setting

```
"no-namespace": [true, "allow-declarations"]
```

That would disallow namespaces in real code, but keep it valid in type definitions like these.